network-api/networkapi/templates/partials/banner.html

```diff
@@ -1,8 +1,16 @@
 {% load settings_value i18n static %}
 
-{% with wrapper_class="tw-py-7" %}
-<div class="donate-banner tw-bg-black {{wrapper_class}} print:tw-hidden tw-w-full tw-hidden">
-    <a href="#" class="p-4 tw-box-content banner-close tw-cursor-pointer tw-absolute tw-right-0 tw-top-0 tw-group" aria-label="{% trans "Close Banner" %}">
+{% with wrapper_class="tw-py-7" background_class="tw-bg-cover tw-bg-center" btn_class="tw-text-white tw-border-black tw-bg-blue-40 hover:tw-bg-blue-80 tw-shadow-black"%}
+<div class="donate-banner {{background_class}} tw-bg-[url(../_images/mozilla-donate-dear-internet-mobile.png)] large:tw-bg-[url(../_images/mozilla-donate-dear-internet.png)] {{wrapper_class}} print:tw-hidden tw-w-full tw-hidden">
+    <a href="#" 
+       class="p-4
+              tw-box-content
+              banner-close
```

Comment:
**Question** Do we still need `banner-close`?

Reply:
used for JS file targets